### PR TITLE
Write new CLI reference

### PR DESCRIPTION
**Pull request by tenodi**

### CODE REVIEW

File: _posts/1-4-1-CLI.md [hunk 1]
```diff
@@ -6,123 +6,322 @@ section: Extensions
 ---
 
 # Shoutem CLI
-Shoutem Command Line Interface (CLI) is a tool to manage your extensions. By using it you can create extensions, generate code snippets, upload extensions to your Shoutem account and install them on your applications. Here you can find a list of available commands. 
+Shoutem Command Line Interface (CLI) is a tool that helps you build extensions. Using it, you can create extensions, generate code snippets, upload extensions to your Shoutem account and install them on your applications.
```
Comment (ikatun) [inline _posts/1-4-1-CLI.md]: 'CLI allows you to create extensions, ...' or 'It allows you to...'
This sounds like a Croatian sentence :)

**Comment by ikatun** [comment]
Looks good

File: _posts/1-4-1-CLI.md [hunk 2]
```diff
@@ -6,123 +6,322 @@ section: Extensions
...
+
+Each Shoutem user can become Shoutem developer. If you still are not a developer, after you entered correct credentials, you'll be prompted a new developer name.
+
+```ShellSession
+Enter new developer name:
+Name: david
+You are registered as `david`!
+```
+
+- ##### shoutem whoami
```
Comment (tenodi) [inline _posts/1-4-1-CLI.md]: Yea, but it's not UX friendly to request to use `login` for this. We need to implement `whoami` (npm has it too).

File: _posts/1-4-1-CLI.md (hunk 1 above)
Comment (tenodi) [inline _posts/1-4-1-CLI.md]: 👍

**Pull request merged**